packages/flutter_tools/test/general.shard/features_test.dart

```diff
@@ -411,5 +411,46 @@ void main() {
       expect(swiftPackageManager.stable.enabledByDefault, false);
       expect(swiftPackageManager.stable.available, true);
     });
+
+    group('Swift Package Manager app migration feature', () {
+      test('availability and default enabled', () {
+        expect(swiftPackageManagerMigration.master.enabledByDefault, false);
+        expect(swiftPackageManagerMigration.master.available, true);
+        expect(swiftPackageManagerMigration.beta.enabledByDefault, false);
+        expect(swiftPackageManagerMigration.beta.available, true);
+        expect(swiftPackageManagerMigration.stable.enabledByDefault, false);
+        expect(swiftPackageManagerMigration.stable.available, true);
+      });
+
+      test('requires Swift Package Manager feature', () {
+        platform.environment = <String, String>{
+          'SWIFT_PACKAGE_MANAGER': 'false',
```

Comment:
This environment variable name is a bit generic and is susceptible to collisions with other tooling. Ideally we would rename this to `FLUTTER_SWIFT_PACKAGE_MANAGER`. 

However, renaming this would break anyone that's set this environment variable to `true`. To minimize disruption, I will rename this environment variable in the same release that turns on SwiftPM by default.

Tracked by https://github.com/flutter/flutter/issues/159121